data/json/daschland.json

```diff
@@ -0,0 +1,300 @@
+{
+    "$schema": "https://raw.githubusercontent.com/dasch-swiss/dsp-meta-svc/main/docs/services/metadata/schema-metadata.json",
+    "project": {
+        "__type": "Project",
+        "status": "Ongoing",
+        "howToCite": "Alice in DaSCHland Project Team (2025). Alice in DaSCHland Dataset [Database]. DaSCH. https://ark.dasch.swiss/ark:/72163/1/0854.",
+        "teaserText": "Alice in DaSCHland whisks you away on a whimsical digital journey where data and metadata spark fantastical adventures - showcase project from DaSCH!",
+        "datasets": [
+            "http://ns.dasch.swiss/repository#dsp-0854-dataset-000",
+            "http://ns.dasch.swiss/repository#dsp-0854-dataset-001"
+        ],
+        "contactPoint": "http://ns.dasch.swiss/repository#dsp-0854-organization-000",
+        "description": {
+            "fr": "Alice in DaSCHland is the showcase project from DaSCH that reimagines Alice’s adventures through a digital lens. The project blends literary elements with advanced data management systems, offering users a unique exploration of data curation, visualization, and retrieval in a surreal narrative landscape.\nAlice in DaSCHland est le projet vitrine de DaSCH qui réimagine les aventures d'Alice à travers une perspective numérique. \nLe projet mélange des éléments littéraires avec des systèmes de gestion de données avancés, offrant aux utilisateurs une exploration unique de la préservation, de la visualisation et de la recherche de données dans un paysage narratif surréaliste."
+        },
```

Comment:
this is marked as a single, french description, but it should be english and french